src/dds_simple.h

```diff
@@ -244,93 +244,151 @@ class DDSSimpleManager : public DDSManager
     // This function waits until it finds one (or more) Subscriber of topic T OR secondsToWait seconds expires.
     // Useful when making sure your messages are being Published on a certain topic.
     template <class T>
-    bool WaitForSubscriber(int secondsToWait = 2)
+	bool WaitForSubscriber(std::chrono::milliseconds timeToWait = std::chrono::seconds(2))
```

Comment:
Can you replace the tabs with spaces?

Reply:
Formatted the whole file with tabs.

Reply:
I don't really have anything against using tabs in general, but the rest of the repo is using spaces, so I think this file should be the same.